arch/arm/configs/tsimx6ul_defconfig

```diff
@@ -857,26 +714,19 @@ CONFIG_DRM_IMX=y
 CONFIG_DRM_IMX_PARALLEL_DISPLAY=y
 CONFIG_DRM_MXSFB=y
 CONFIG_FB_MODE_HELPERS=y
+# CONFIG_FB_MX3 is not set
 CONFIG_FB_ST7565P=m
-CONFIG_DRM_FBDEV_EMULATION=y
 CONFIG_LCD_CLASS_DEVICE=y
+CONFIG_LCD_PLATFORM=y
 CONFIG_BACKLIGHT_CLASS_DEVICE=y
-CONFIG_BACKLIGHT_PWM=m
-CONFIG_BACKLIGHT_GPIO=m
-CONFIG_SOUND=y
-CONFIG_SND=y
-# CONFIG_SND_ARM is not set
-# CONFIG_SND_SPI is not set
-CONFIG_SND_USB_AUDIO=m
```

Comment:
USB audio support not removed from every platform? I think we should keep this in as a module just in case. But if not, should be removed from everything.

Reply:
Added to our requirements.